src/vs/workbench/browser/media/style.css

```diff
@@ -76,9 +76,9 @@ body.web {
 	cursor: pointer;
 }
 
-.monaco-workbench a {
-	text-decoration: none;
-}
+/* .monaco-workbench a {
+	text-decoration: none; {{SQL CARBON EDIT}} Underline links
```

Comment:
we usually would try not to touch the original content, maybe comment out this way?
```CSS
/* {{SQLCARBON EDIT}} Underline links
.monaco-workbench a {
	text-decoration: none;
}
```
*/

Reply:
Any reason why? From the perspective of a merge conflict it's all going to just be a single conflict anyways so as long as there's a carbon edit somewhere that fulfills the needs from that point of view.

Reply:
And I went there just to indicate that it was that one specifically. Don't really care either way but curious if you had a specific reason for wanting it the other way

Reply:
yeah, for this one probably it is ok.